src/ontology/uberon-edit.obo

```diff
@@ -224246,6 +224246,119 @@ intersection_of: part_of UBERON:0001987 ! placenta
 relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
 property_value: dcterms-date 2022-06-27T09:30:52Z xsd:dateTime
 
+[Term]
+id: UBERON:8480009
+name: perianal space
+def: "The virtual spaces circumscribed by the mucosa lining of the anal canal and the muscular structures of the sphincter complex." [http://www.endoscopy-colon-explorer.com/anatomy-of-the-anal-region/]
+is_a: UBERON:0000464 ! anatomical space
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+relationship: part_of UBERON:0001353 ! anal region
+property_value: dcterms-date 2022-07-05T10:24:30Z xsd:dateTime
+
+[Term]
+id: UBERON:8480010
+name: subcutaneous perianal space
+def: "A space located below the dentate line. Its borders are the skin of the anal verge below and the internal sphincter above." [http://www.endoscopy-colon-explorer.com/anatomy-of-the-anal-region/]
+is_a: UBERON:8480009 ! perianal space
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-07-05T10:36:29Z xsd:dateTime
+
+[Term]
+id: UBERON:8480011
+name: submucous perianal space
+def: "A space located above the dentate line. It is delimited by the internal sphincter and by the anoderm." [http://www.endoscopy-colon-explorer.com/anatomy-of-the-anal-region/]
```

Comment:
The first sentence of the def should always be the differentia - see "Genus-differentia definitions are your friend" section of https://douroucouli.wordpress.com/2019/07/08/ontotip-write-simple-concise-clear-operational-textual-definitions/

Reply:
@shawntanzk Here you mean the same as your first comment? That the definition should be:
"A **perianal** space located above the dentate line. It is delimited by the internal sphincter and by the anoderm."

Reply:
@shawntanzk the 1st sentence says "below" and the 2nd "above". Should I make it more different?

Reply:
oh whoops! my bad hahaha, sorry I totally saw it as the same thing for some reason, I blame jetlag >.< no worries then

Reply:
@shawntanzk OK I think I got that I should add the "delimited by" to the first senntence

Reply:
@paulacalipho, I think it makes sense to incorporate the boundaries of the anatomical space in the first sentence of the definition.

Reply:
@bvarner-ebi  I incorporated the boundaries in the first sentence. Thanks

Reply:
@shawntanzk @bvarner-ebi I removed this term to align to the classification of spaces described in https://www.ncbi.nlm.nih.gov/books/NBK535387/